wallet/txauthor/cprng.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package txauthor
+
+import (
+	"crypto/rand"
+	"encoding/binary"
+	mrand "math/rand"
+	"sync"
+)
+
+// cprng is a cryptographically random-seeded math/rand prng.  It is seeded
+// during package init.  Any initialization errors result in panics.  It is safe
+// for concurrent access.
+var cprng = cprngType{}
```

Comment:
If you feel strongly about that then please make a btcwallet pr with the change, or change it after the sync is committed to dcrwallet.  Changes that don't belong to either merge parent result in what's known as an "evil merge" and I won't be adding them.